package/rules/sass.js

```diff
@@ -1,6 +1,6 @@
 const getStyleRule = require('../utils/get_style_rule')
 
-module.exports = getStyleRule(/\.(scss|sass)$/i, false, [
+module.exports = getStyleRule(/\.(scss|sass)(\.erb)?$/i, false, [
```

Comment:
Looks great, did you test the regex? `/\.(scss|sass)?(\.erb)?$/` (optional .erb ext) I am no expert but feel like this is correct.